### PR TITLE
Added homework_11_141222 (nested, inner, static, local, anonymous cla…

### PR DESCRIPTION
**Pull request by kate-lvlup**
Hi Dima,
Hope you have electricity!
Could you check my homework11 (part about nested classes)?

Thanks in advance!

### CODE REVIEW

**Comment by kate-lvlup** [comment]
Hi Dima, I've added a commit with validatePassword() via regex to RegistrationApp

File: homework_10_proj_101222/src/RegistrationApp.java [hunk 1]
```diff
@@ -84,28 +48,132 @@ public static void main(String[] args) {
...
+        return password.matches("[a-zA-Z0-9]{4,}");
+    }
+
+    public  static  boolean validatePassword(String password){
+        return password.matches("^(?=.*[!@#$%])(?=\\S+$).{4,}$");
+    }
+
+
+//    //TODO RegEx
+//    public static boolean validatePassword(String password) {
```
Comment (YeaaaaH) [inline homework_10_proj_101222/src/RegistrationApp.java:141]: delete unused code

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
need fix

File: homework_10_proj_101222/src/RegistrationApp.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_10_proj_101222/src/RegistrationApp.java:141]: Fixed.

**Review by YeaaaaH — APPROVED** [review]

**Comment by YeaaaaH** [comment]
resolve and merge

**Pull request merged**